db/migrations/20240622181744_AddSessions.up.sql

```diff
@@ -0,0 +1,9 @@
+-- Add migration script here
+CREATE TABLE
+  Sessions (
```

Comment:
This is for User Auth.

When you go to a page we will create a Session row for you and add the `session_id` as a private Cookie on your browser.

We will then use this session_id cookie to find your Session. This is how we will know a user is logged in! They will have a session with a non-null `user_id` when they are signed in